browser_extension/js/main.js

```diff
@@ -454,46 +342,41 @@ chrome.storage.sync.get({
         });
 
         // Socket events
-
-        // Whenever the server emits 'login', log the login message
+        // Whenever the server emits 'login', log the message
         socket.on('login', function (data) {
           connected = true;
           // Display the welcome message
           var message = "Welcome to Chaddon – ";
           log(message, {
             prepend: true
           });
-          addParticipantsMessage(data);
+
         });
 
         // Whenever the server emits 'new message', update the chat body
         socket.on('new message', function (data) {
           if (data.room == currentChannel) {
             addChatMessage(data);
-
-            // if (payload.username != data.username) {
```

Comment:
add back in if (payload.username != data.username)  condition here so the user cant trigger their own notifications.